concreate/builders/osbs.py

```diff
@@ -0,0 +1,203 @@
+import logging
+import os
+import shutil
+import subprocess
+
+from concreate.builder import Builder
+from concreate.errors import ConcreateError
+
+logger = logging.getLogger('concreate')
+
+
+class OSBSBuilder(Builder):
+    """Class representing OSBS builder."""
+
+    def prepare(self, descriptor):
+        """Prepares dist-git repository for OSBS build."""
+
+        repository_key = descriptor.get('osbs', {}).get('repository', {})
+        repository = repository_key.get('name')
+        branch = repository_key.get('branch')
+
+        if not (repository and branch):
+            raise ConcreateError("Osbs builder needs repostiory and branch provided!")
+
+        self.dist_git_dir = os.path.join(os.path.expanduser('~/.concreate.d'),
```

Comment:
I'm a bit unsure about this directory name, but I think it could be left.

Reply:
yes. Im not sure too - but i didn't want to change .concreate to directory in this PR